integration-tests/cucumber-tests-platform-smartmetering/src/test/resources/features/osgp-adapter-ws-smartmetering/bundle/BundledGetGsmDiagnostic.feature

```diff
@@ -0,0 +1,33 @@
+@SmartMetering @Platform @GetGsmDiagnostic
+Feature: SmartMetering Bundle - Get Gsm Diagnostic
+  As a grid operator
+  I want to be able to get the GSM Diagnostic of a smart meter
+
+  Scenario: Get the GSM Diagnostic of a E-Meter
+    Given a bundle request
+      | DeviceIdentification | TEST1027000000001 |
+    And the bundle request contains a get gsm diagnostic action
+    And a dlms device
+      | DeviceIdentification | TEST1027000000001 |
+      | DeviceType           | SMART_METER_E     |
+      | CommunicationMethod  | GPRS              |
+      | Protocol             | SMR               |
+      | ProtocolVersion      |               5.1 |
+      | Port                 |              1027 |
+    When the bundle request is received
+    Then the bundle response should contain a get gsm diagnostic response with values
+      | operator                  | Utility Connect          |
+      | modemRegistrationStatus   | REGISTERED_ROAMING       |
+      | circuitSwitchedStatus     | INACTIVE                 |
+      | packetSwitchedStatus      | GPRS                     |
+      | cellId                    | 77                       |
+      | locationId                | 2230                     |
+      | signalQuality             | MINUS_87_DBM             |
+      | bitErrorRate              | RXQUAL_6                 |
+      | mobileCountryCode         |                       66 |
+      | mobileNetworkCode         |                      204 |
+      | channelNumber             |                      107 |
+      | numberOfAdjacentCells     |                        2 |
+      | adjacentCellId            | 93                       |
+      | adjacentCellSignalQuality | MINUS_91_DBM             |
+#      | captureTime               | 2021-04-13T08:45:00.000Z |
```

Comment:
This is commented because the captureTime is not read by the command executer. If we don't comment it, the test will fail. Btw still waiting to get more information about the captureTime. On Monday somebody from the meterlab will check what the meters over there return for this attribute. Then we can decide what we do.

Reply:
Added an explanation